vhpidirect/quickstart/logicvector/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+static const char HDL_LOGIC_STATE[] = { 'U', 'X', '0', '1', 'Z', 'W', 'L', 'H', '-'};
+
+char logic_vec[9];
+int logic_vec_size = 9;
```

Comment:
If both are defined explicitly, what about putting the int first and `char logic_vec[logic_vec_size];`?

Furthermore, do we want to define `logic_vec_size`? Can we hardcode `sizeof(logic_vec)/sizeof(char)`, maybe through a `#define`? Compared to earlier (simpler) examples, this would show that multiple C features can be used.

Reply:
Closest is:
`#define SIZE_LOGIC_VEC_A (sizeof(logic_vec_A)/sizeof(char))`

Quite a neat trick you pointed me to, thanks